app/views/register-with-a-gp/register-without-signin.html

```diff
@@ -30,12 +30,41 @@ <h1 class="heading-xlarge">
       })
     }}
 
-    {% include "includes/forms/gp-practice.html" with {
-      label: "Current GP practice or doctor"
-    } %}
+    {{
+      form_macros.address_fields({
+        id: 'address-1',
+        label: 'Address',
+        hint: 'As you’d write it on a envelope'
+      })
+    }}
+
+    {{
+      form_macros.text_field({
+        id: 'email',
+        label: 'Email address'
+      })
+    }}
+
+    {{
+      form_macros.text_field({
+        id: 'telephone',
+        label: 'Telephone number (optional)'
+      })
+    }}
+
+    <div class="form-group">
+      <label class="form-label-bold" for="nhs-number">
+        NHS number (optional)
+        <span class="form-hint">
+          If you know it &mdash; for example 943 476 5919
```

Comment:
Is this NHS number a known example? If it's just random numbers, we should change it to something that we know to be not real.